src/snowflake/snowpark/_internal/analyzer/snowflake_plan.py

```diff
@@ -731,7 +730,7 @@ def large_local_relation_plan(
         source_plan: Optional[LogicalPlan],
         schema_query: Optional[str],
     ) -> SnowflakePlan:
-        temp_table_name = random_name_for_temp_object(TempObjectType.TABLE)
+        temp_table_name = f"temp_name_placeholder_{generate_random_alphanumeric()}"
```

Comment:
isn't that going to impact the df.queries? not just people will see a different table, but is people going to see a non-sense temp table name called "temp_name_placeholder_xxx"?

Reply:
Is large_local_relation_plan it only case we will have internally created query scoped temp table?

Reply:
A more efficient way for those is probably making the temp table session scoped, and let @sfc-gh-jdu 's temp table clean up to handle the table drop off, i guess we can do this as future improvement